src/main/java/sk/freemap/gpxAnimator/ui/GeneralSettingsPanel.java

```diff
@@ -89,52 +91,52 @@ public GeneralSettingsPanel() {
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("JPEG Image Frames", "jpg"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("PNG Image Frames", "png"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("H.264 Encoded Video Files (*.mp4, *.mov, *.mkv)", "mp4", "mov", "mkv"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("MPEG-1 Encoded Video Files (*.mpg)", "mpg"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("MPEG-4 Encoded Video Files (*.avi)", "avi"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("MS MPEG-4 Encoded Video Files (*.wmv, *.asf)", "wmv", "asf"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("Theora Encoded Video Files (*.ogv)", "ogv"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("FLV Encoded Video Files (*.flv)", "flv"));
+				outputFileChooser.addChoosableFileFilter(new FileNameExtensionFilter("RV10 Encoded Video Files (*.rm)", "rm"));
+				return Type.SAVE;
+			}
 
-					@Override
-					protected String transformFilename(final String filename) {
-						if ((filename.toLowerCase().endsWith(".png") || filename.toLowerCase().endsWith(".jpg"))
-								&& String.format(filename, 100).equals(String.format(filename, 200))) {
-							final int n = filename.lastIndexOf('.');
-							return filename.substring(0, n) + "%08d" + filename.substring(n);
-						} else {
-							return filename;
-						}
-					}
-				};
+			@Override
+			protected String transformFilename(final String filename) {
+				if ((filename.toLowerCase().endsWith(".png") || filename.toLowerCase().endsWith(".jpg"))
+						&& String.format(filename, 100).equals(String.format(filename, 200))) {
+					final int n = filename.lastIndexOf('.');
+					return filename.substring(0, n) + "%08d" + filename.substring(n);
+				} else {
+					return filename;
+				}
+			}
+		};
 
-				outputFileSelector.setToolTipText(Option.OUTPUT.getHelp());
-				final GridBagConstraints gbc_outputFileSelector = new GridBagConstraints();
-				gbc_outputFileSelector.fill = GridBagConstraints.BOTH;
-				gbc_outputFileSelector.insets = new Insets(0, 0, 5, 0);
-				gbc_outputFileSelector.gridx = 1;
-				gbc_outputFileSelector.gridy = 0;
-				add(outputFileSelector, gbc_outputFileSelector);
+		outputFileSelector.setToolTipText(Option.OUTPUT.getHelp());
+		final GridBagConstraints gbc_outputFileSelector = new GridBagConstraints();
+		gbc_outputFileSelector.fill = GridBagConstraints.BOTH;
+		gbc_outputFileSelector.insets = new Insets(0, 0, 5, 0);
+		gbc_outputFileSelector.gridx = 1;
+		gbc_outputFileSelector.gridy = 0;
+		add(outputFileSelector, gbc_outputFileSelector);
 
-				outputFileSelector.addPropertyChangeListener("filename", propertyChangeListener);
+		outputFileSelector.addPropertyChangeListener("filename", propertyChangeListener);
```

Comment:
There are so many code formatting changes in this file, I really have problems identifying your code changes in these parts! Please revert all your code formatting changes, so your PR only contains the changed code. If you want to change the code formatting style, we can discuss it and do it in a separate commit and PR.